### PR TITLE
misc improvements and bug fixes

### PR DESCRIPTION
**Pull request by kkraune**
new console endpoint name

use mynamespace instead of just "music" do de-confuse what the id is built of 

make bundle load using correct name

make queries work

### CODE REVIEW

File: album-recommendation-java/README.md [hunk 1]
```diff
@@ -104,3 +108,9 @@ $ openssl req -x509 -nodes -days 14 -newkey ec:<(openssl ecparam -name prime256v
     -   as arguments to `mvn` like `-DdataPlaneCertificateFile=$HOME/clients.pem`, or
     -   as VM options for the JUnit tests in the IDEA, like `-DdataPlaneCertificateFile=$HOME/clients.pem`. 
     This also applies to the `dataPlanePrivateKeyFile`, and the `privateKeyFile`.
+
+
+<!-- Troubleshooting notes
```
Comment (jonmv) [inline album-recommendation-java/README.md]: What's this?

File: album-recommendation-java/pom.xml [hunk 2]
```diff
@@ -6,7 +6,7 @@
                              http://maven.apache.org/xsd/maven-4.0.0.xsd">
     <modelVersion>4.0.0</modelVersion>
     <groupId>com.yahoo.example</groupId>
-    <artifactId>album-recommendation-java</artifactId>
+    <artifactId>album-recommendation-java</artifactId>  <!-- Note: When changing this, also change bundle names in services.xml -->
```
Comment (jonmv) [inline album-recommendation-java/pom.xml]: Aha. So, did it stop the logserver too?

**Comment by jonmv** [comment]
Check out CD console — user keys are on their way out!

File: album-recommendation-java/README.md [hunk 3]
```diff
@@ -70,25 +69,30 @@ $ openssl req -x509 -nodes -days 14 -newkey ec:<(openssl ecparam -name prime256v
...
-      $ENDPOINT/document/v1/music/music/docid?wantedDocumentCount=100
+      $ENDPOINT/document/v1/mynamespace/music/docid?wantedDocumentCount=100
     ```
     
 1.  Search documents:
     ```sh
-    $ curl --cert $HOME/clients.pem --key $HOME/key.pem \
-      $ENDPOINT/search/?query=bad
+    $ curl --cert src/main/application/security/clients.pem --key $HOME/key.pem \
+    "$ENDPOINT/search/?ranking=rank_albums&amp;yql=select%20%2A%20from%20sources%20%2A%20where%20sddocname%20contains%20%22music%22%3B&amp;ranking.features.query(user_profile)=%7B%7Bcat%3Apop%7D%3A0.8%2C%7Bcat%3Arock%7D%3A0.2%2C%7Bcat%3Ajazz%7D%3A0.1%7D"
```
[the remark this answers is not in the document]
Comment (kkraune) [reply album-recommendation-java/README.md]: yes - it gets right in the rendered html ...

File: album-recommendation-java/README.md (hunk 1 above)
Comment (kkraune) [reply album-recommendation-java/README.md]: my own notes. first time you deploy, if the container does not get up (and it does not if you mess up bundle name), there is not way to find this log, as the instance is not set up as it never succeeds. I think. must confirm again. kept this not for myself for now

File: album-recommendation-java/pom.xml (hunk 2 above)
Comment (kkraune) [reply album-recommendation-java/pom.xml]: I need to check this again

**Pull request merged**